Task 9: Delete comments

sausagroll90 opened the pull request:
Users can now delete comments they have posted.
I have used createContext and useContext to provide a context for the current user, which is displayed at the top of the screen.

pippaamy reviewed (commented):
this is great, well done :)

sausagroll90 committed:
Feature: Add delete button to comments for currently signed-in user

Users can now delete comments they have posted.
I have used createContext and useContext to provide a context for the current user, which is displayed at the top of the screen.

hevdevs commented on src/components/CommentCard.jsx:
```diff
@@ -1,10 +1,28 @@
+
+export default function CommentCard({ comment, date, fetchComments }) {
+  const { user } = useContext(UserContext);
+  const [isDeleteButtonDisabled, setIsDeleteButtonDisabled] = useState(false);
```
Nice!

hevdevs reviewed (commented):
Don't forget to implement loading states or similar for improved UX experience - some communication to users when requests are in progress vs failed. Can be confusing as to what's going on to users otherwise! Looking great otherwise Dom, nice work!
